meta-ros2/classes/ros_ament_cmake.bbclass

```diff
@@ -38,3 +40,8 @@ EXTRA_OECMAKE:prepend:class-native = "\
     -DCMAKE_PREFIX_PATH='${ros_prefix}' \
     -DCMAKE_INSTALL_PREFIX:PATH='${ros_prefix}' \
 "
+
+EXTRA_OECMAKE:prepend:class-nativesdk = "\
+    -DCMAKE_PREFIX_PATH='${STAGING_DIR_NATIVE}/opt/ros/${ROS_DISTRO};${STAGING_DIR_NATIVE}${ros_prefix};${STAGING_DIR_NATIVE}${prefix}' \
```

Comment:
Isn't `${STAGING_DIR_NATIVE}/opt/ros/${ROS_DISTRO}` and `${STAGING_DIR_NATIVE}${ros_prefix}` the same? (except, of course, in situations where `ros_prefix` is customized - like in my build).

```suggestion
    -DCMAKE_PREFIX_PATH='${STAGING_DIR_NATIVE}${ros_prefix};${STAGING_DIR_NATIVE}${prefix}' \
```

Reply:
Sure seem like a problem.  I will try on my desktop

Reply:
See https://github.com/ros/meta-ros/pull/1241/commits/b99fe2b6e538bce8e1c49485c9a75032f870b255